RSignatureVerifier/APIClient.swift

```diff
@@ -0,0 +1,71 @@
+protocol SessionProtocol {
+    func startTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void)
+}
+
+extension URLSession: SessionProtocol {
+    func startTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) {
+        dataTask(with: request) { (data, response, error) in
+            completionHandler(data, response, error)
+        }.resume()
+    }
+}
+
+internal struct Response {
+    let object: Parsable
+    let httpResponse: HTTPURLResponse
+
+    init(_ object: Parsable, _ response: HTTPURLResponse) {
+        self.object = object
+        self.httpResponse = response
+    }
+}
+
+internal class APIClient {
```

Comment:
Is it internal by default right?
Can APIClient be final class?

Reply:
final done. Internal is default but I remember some conversation that we decided (for IAM) to use explicit `internal` to visibly distinguish what is public and what is not. I don't have any strong opinion on that. The code is taken from Remote Config repo and I think leaving `internal` will make our public SDKs more consistent

Reply:
You can leave it as internal for now